index.js

```diff
@@ -18,7 +18,9 @@ var DEFAULTS = {
         addNotifyTrigger: 'add_notify_trigger_to_table',
         removeNotifyTrigger: 'remove_notify_trigger_to_table'
     },
-    operationEvents: true
+    operationEvents: true,
+    checkUpdates: true, // only notify if update changes record, can be set to false if record comparison is expensive
+    sendRecordId: false // send only record IDs, true - to send column 'id', string to send column with a given name
```

Comment:
I thought about it. I even started... I didn't do it as I thought it would mean a different trigger function for each table (as the list of columns is table specific) and it seemed like too much of a hassle. But now I realise that the trigger can pass the list of columns to trigger function as parameter, so the trigger function will be the same. So it is a bit easier. The columns option should be in subscribe method though, not in constructor. I thought dataColumns and oldDataColumns